ocs_ci/helpers/dr_helpers.py

```diff
@@ -799,16 +803,34 @@ def wait_for_replication_resources_creation(
         logger.error(error_msg)
         raise TimeoutExpiredError(error_msg)
 
+    if mix_workload:
+        resource_map = {
+            "rbd": (constants.VOLUME_REPLICATION, get_vr_count),
+            "cephfs": (constants.REPLICATION_SOURCE, get_replicationsources_count),
+        }
+
+        for key, vr_count in mix_workload_data.items():
+            if key not in resource_map:
+                logger.warning(
+                    f"Unsupported storage type '{key}' in mix_workload. Skipping..."
+                )
+                continue
+
+            resource_kind, count_function = resource_map[key]
+            wait_for_resource_creation_and_state(
```

Comment:
In `wait_for_resource_creation_and_state`, there is no validation for replicationsources being done. So you we can skip passing cephfs args to that function